### ISSUE TITLE
Unable to get the Sonar PDF on mail

### ISSUE THREAD

**Issue by madhurilalit** [SonarQubeCommunity/sonar-pdf-report#31]
Hi,

I have configured my sonar project view to receive a sonar PDF report however i am still not getting the same on email. Below are the steps i performed:

1. I have created a view where I have included my 6 projects. I gave execute permission on view not on individual project. 
2. I have set Executive Report option to come daily and provided email id's to receive the pdf
3.  I have set the Subscribe button which says "You are receiving PDF reports by email every day."

Please suggest what is missing. Also the report doesn't include much details about the sonar code analysis. Is it possible to include below parameters:

- Test Coverage
- Unresolved Issues

Please resolve this issue asap as i am not getting much help for web on the same.

**Comment by madhurilalit** [comment]
Gentle followup!!!

**Comment by madhurilalit** [comment]
Gentle followup!!! Could you please respond!

**Comment by madhurilalit** [comment]
Gentle followup!!! Could you please respond!

**Comment by designerferro** [comment]
What is the SonarQube version you are testing on?

**Comment by madhurilalit** [comment]
Version 5.6.2

**Comment by designerferro** [comment]
Have you changed the API call for issues unordered to comply with the new API version? Where is it published?

**Comment by madhurilalit** [comment]
Can you explain how to change API call for issues unordered to comply with the new API version? What do mean by where is it published? Can you please explain in simpler terms since I am new to Sonar.

**Comment by designerferro** [comment]
You can find more information about extending SonarQube with Java API at https://docs.sonarqube.org/plugins/servlet/mobile#content/view/1441915

MVN repository here https://mvnrepository.com/artifact/org.codehaus.sonar/sonar-java-api

API changes here https://docs.sonarqube.org/plugins/servlet/mobile#content/view/1442007

**Comment by madhurilalit** [comment]
Let me follow the above links and meanwhile don't close the ticket.

**Comment by designerferro** [comment]
Oh boy, why you can't understand that I'm not the programmer. Hell, I'm not even one. I just posted some fixes for a problem when this was working in versions prior to 5.0.

**Comment by madhurilalit** [comment]
Do you mean in Sonarqube Version 5.6.2, PDF email functionality is not working?

**Comment by madhurilalit** [comment]
Kindly answer my query that whether PDF functionality works in Sonarqube Version 5.6.2 or not. Also please confirm is it possible to include below parameters in sonar report:

- Test Coverage
- Unresolved Issues

**Comment by designerferro** [comment]
I don't know. We stoped using this plugin 2 months or so ago.

I don't remember ever seeing the option to send the PDF or the test coverage on the reports.

SounarSource has a payed version of SonarQube where this functions are included.

**Comment by designerferro** [comment]
The PDF did come with unsolved issues, but since version 5.0 and up had the issues API changed, this stoped working.
